example-library/src/main/resources/application-example.yaml

```diff
@@ -59,11 +59,17 @@ population:
       rootId: ${default.rootId}
       connectionDetail: ${default.connectionDetail}
       attributes:
-        type: employee
+        type: uk.gov.gchq.palisade.example.hrdatagenerator.types.Employee
         serialisedFormat: avro
     - resourceId: ../Palisade-examples/resources/data/employee_file1.avro
       rootId: ${default.rootId}
       connectionDetail: ${default.connectionDetail}
       attributes:
-        type: employee
+        type: uk.gov.gchq.palisade.example.hrdatagenerator.types.Employee
         serialisedFormat: avro
+
+  serialiserProvider: std
+  serialisers:
+    - flavourFormat: avro
+      flavourType: uk.gov.gchq.palisade.example.hrdatagenerator.types.Employee
+      serialiserClass: uk.gov.gchq.palisade.data.serialise.AvroSerialiser
```

Comment:
Can we move the types and formats to the `default` field like connectionDetail so they aren't duplicated over the file like this? It'll just be a yaml change, won't need any java changes

Reply:
ie. reference them by ${default.type} and ${default.format}

Reply:
This has been addressed and the example still runs without any issues.